headless/headless-commerce/headless-commerce-admin-inventory-impl/src/main/java/com/liferay/headless/commerce/admin/inventory/internal/resource/v1_0/WarehouseItemResourceImpl.java

```diff
@@ -257,6 +268,8 @@ public WarehouseItem postWarehousIdWarehouseItem(
 		return warehouseItems;
 	}
 
+	private static int _defaultIncrementDays = 30;
```

Comment:
Ideally this field would be `final`, which then would change the naming convention to _ALL_CAPS.